test/test-csp-inline.html

```diff
@@ -0,0 +1,61 @@
+<!doctype html>
+<html>
+  <head>
+    <link rel="stylesheet" type="text/css" href="../bower_components/qunit/qunit/qunit.css"/>
+    <meta http-equiv="Content-Security-Policy" content="script-src 'self' 'nonce-abc';">
+  </head>
+  <body>
+
+    <h1 id="qunit-header">SystemJS Test Suite</h1>
+
+    <h2 id="qunit-banner"></h2>
+    <div id="qunit-testrunner-toolbar"></div>
+    <h2 id="qunit-userAgent"></h2>
+    <ol id="qunit-tests"></ol>
+    <div id="qunit-test-area"></div>
+
+    <script src="../dist/system.src.js" type="text/javascript"></script>
```

Comment:
The CSP build will disallow any inline scripts. Because "nonce" requires inline to actually do anything, just use the non-csp version.